mathics/builtin/numbers/linalg.py

```diff
@@ -194,21 +194,33 @@ class Inverse(Builtin):
      : The matrix {{1, 0}, {0, 0}} is singular.
      = Inverse[{{1, 0}, {0, 0}}]
 
-    >> Inverse[{{1, 0, 0}, {0, Sqrt[3]/2, 1/2}, {0,-1 / 2, Sqrt[3]/2}}]
-    = {{1, 0, 0}, {0, Sqrt[3] / 2, -1 / 2}, {0, 1 / 2, Sqrt[3] / 2}}
     """
 
-    messages = {"sing": "The matrix `1` is singular."}
+    messages = {
+        "sing": "The matrix `1` is singular.",
+        "matsq": ("Argument `1` at position 1 is not " "a non-empty square matrix."),
```

Comment:
A bit weird style here (parenthesis and 2 strings).

Reply:
Sorry, it started being two lines...

Reply:
OK, it is clean now.